### PR TITLE
added animated presentation and styles

### PR DESCRIPTION
**Pull request by soniaigartua**
added animated presentation Screen and configured styles for layouts

Evidences:
presentationScreen

### CODE REVIEW

File: app/src/main/java/com/example/pps_tudai/mvp/presenter/PresentationPresenter.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+public class PresentationPresenter {
+
+    private final PresentationView presentationView;
+
+    public PresentationPresenter(PresentationView presentationView) {
+        this.presentationView = presentationView;
+    }
+
+    public void startWelcomeAnimation (Animation animation1, Animation animation2) {
+        presentationView.getWelcomeMessage().setAnimation(animation1);
```
Comment (MuroSar) [inline app/src/main/java/com/example/pps_tudai/mvp/presenter/PresentationPresenter.java]: you can put this two lines in a method in the view called initAnimations for example.

File: app/src/main/java/com/example/pps_tudai/mvp/view/PresentationView.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    public void showMainScreen() {
+        if (activityWeak.get() != null) {
+            Intent main = new Intent(activityWeak.get(), MainActivity.class);
+            activityWeak.get().startActivity(main);
+        }
+    }
+
+    public void llega() {
```
Comment (MuroSar) [inline app/src/main/java/com/example/pps_tudai/mvp/view/PresentationView.java]: please use english

File: app/src/main/res/layout/activity_presentation.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+android:layout_height="match_parent"
+android:orientation="vertical"
+tools:context=".activity.MainActivity"
+android:gravity="center"
+android:paddingTop="@dimen/padding_top_enter"
+android:background="@color/background_enter" >
+
+<ImageView
+    android:id="@+id/logo"
+    android:layout_width="300sp"
```
Comment (MuroSar) [inline app/src/main/res/layout/activity_presentation.xml]: 1-. Please no hardcode, use dimens.
2-. Remember that we only use "sp" for textSize

File: app/src/main/res/values/colors.xml [hunk 4]
```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#6F6A69</color>
-    <color name="colorPrimaryDark">#CEA87A</color>
+    <!--<color name="colorPrimary">#6F6A69</color>-->
```
Comment (MuroSar) [inline app/src/main/res/values/colors.xml]: remove comments, pls

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -1,6 +1,11 @@
...
+    <color name="colorPrimary">#878988</color>
+    <!--<color name="colorPrimaryDark">#CEA87A</color>-->
```
Comment (MuroSar) [inline app/src/main/res/values/colors.xml]: same

Comment (soniaigartua) [reply app/src/main/res/values/colors.xml]: done!

File: app/src/main/res/values/colors.xml (hunk 4 above)
Comment (soniaigartua) [reply app/src/main/res/values/colors.xml]: done!

File: app/src/main/res/layout/activity_presentation.xml (hunk 3 above)
Comment (soniaigartua) [reply app/src/main/res/layout/activity_presentation.xml]: done!!!

File: app/src/main/java/com/example/pps_tudai/mvp/view/PresentationView.java (hunk 2 above)
Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/mvp/view/PresentationView.java]: I deleted this method ... I wrote it in order to checked something .... I do not need now.

File: app/src/main/java/com/example/pps_tudai/mvp/presenter/PresentationPresenter.java (hunk 1 above)
Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/mvp/presenter/PresentationPresenter.java]: refactored!

**Pull request merged**